.snippets/code/learn/dapps-list/dapp-directory/nfts.json

```diff
@@ -0,0 +1,2118 @@
+{
+  "projects": [
```

Comment:
JSON files should use 4 spaces. More info here: https://github.com/papermoonio/documentation-style-guide/blob/main/style-guide.md#code-formatting-by-language

Reply:
This still isn't using 4 spaces for the indents